i18n/en.json

```diff
@@ -7,5 +7,17 @@
 	"wikibase-export-name": "Wikibase Export",
 	"wikibase-export-description": "Adds a user-friendly Wikibase export page",
 
-	"special-wikibase-export": "Wikibase Export"
+	"special-wikibase-export": "Wikibase Export",
+
+	"wikibase-export-subjects-heading": "Choose companies",
+	"wikibase-export-subjects-placeholder": "Search companies",
```

Comment:
These should actually be configurable 

![image](https://user-images.githubusercontent.com/146040/201089285-73724457-5263-41c4-a4ee-e1d87eede50b.png)

Reply:
Wouldn't overriding `Mediawiki:wikibase-export-subjects-heading` be sufficient?

Reply:
Maybe yes. We could start with that. When we release the extension it is probably good to change the default to "item" instead of "company". Doesn't really matter for now.